CONTRIBUTING.md

```diff
@@ -69,69 +69,56 @@ please adhere to the following contributing guidelines.
     skipAfter
 ```
 
+## Variables naming conventions
+
+* Variable names are lowercase using chars from `[a-z0-9_]`
+* To somewhat reflect the fact that the syntax for variable usage is different when you define it (using setvar) and when you use it, we propose the following visual distinction:
+  * Capital letters for collection, colon as separator, variable name. E.g: `SecRule TX:foo_bar_variable`
+  * Lowercase letters for collection, dot as separator, variable name. E.g: `setvar:tx.foo_bar_variable`
+
 ## Rules compliance with each Paranoia Level (PL)
 
-Rules in the CRS are organized in Paranoia Levels, which are described below.
+Rules in the CRS are organized in Paranoia Levels, which allows you to choose the desired level of rule checks.
 
```

Comment:
Add a remark / link to the crs-setup.conf.example file where the PL mechanism is introduced and explained in detail. What we see here is only a rule of thumb.